File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/GSRSlot.java
package com.pennapps.labs.pennmobile.classes;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by MikeD on 10/9/2017.
 */

//class that represents an available session for a given GSR room
public class GSRSlot {

    @SerializedName("available")
    @Expose
    public boolean available;

    @SerializedName("start")
    @Expose
    public String startTime;

    @SerializedName("end")
    @Expose
    public String endTime;


    public boolean isAvailable() {
        return available;
    }

    public String getStartTime() {
        return startTime;
    }

    public String getEndTime() {
        return endTime;
    }
}

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/GSRSlot.kt
package com.pennapps.labs.pennmobile.classes

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

// class that represents an available session for a given GSR room

/**
 * Created by MikeD on 10/14/2018.
 */

class GSRSlot {
    @SerializedName("available")
    @Expose
    var isAvailable: Boolean = true

    @SerializedName("start_time")
    @Expose
    var startTime: String? = null

    @SerializedName("end_time")
    @Expose
    var endTime: String? = null
}
